subsys/bluetooth/shell/bt.c

```diff
@@ -526,6 +528,36 @@ static int cmd_init(int argc, char *argv[])
 	return 0;
 }
 
+static int cmd_hci_cmd(int argc, char *argv[])
+{
+	u16_t op;
+	struct net_buf *buf = NULL;
+	int err;
+
+	if (argc < 2) {
+		return -EINVAL;
+	}
+
+	op = strtoul(argv[1], NULL, 16);
```

Comment:
I'd make this command accept ocf and ogf for convenience.

Reply:
I though this would be simpler but perhaps doing in ocf/ogf is clearer.